Library/Formula/libwebm.rb

```diff
@@ -11,13 +11,17 @@ class Libwebm < Formula
     sha256 "3e2c2a2fd1854c158c33bd6667a2e0ae17838eb88a0f839ee48316f59d105c7e" => :mountain_lion
   end
 
-  def install
-    system "make"
+  depends_on "cmake" => :build
 
-    lib.install "libwebm.a"
+  def install
+    mkdir "macbuild" do
+      system "cmake", *std_cmake_args, ".."
```

Comment:
`system "cmake", "..", *std_cmake_args` is the standard here and avoids such problems.